server/users.ts

```diff
@@ -602,7 +602,7 @@ export class User extends Chat.MessageContext {
 		}
 		return this.group + this.name;
 	}
-	getIdentityWithStatus(roomid: string = '') {
+	getIdentityWithStatus(roomid: RoomID = '' as RoomID) {
```

Comment:
```suggestion
	getIdentityWithStatus(roomid = '' as RoomID) {
```

You'd think TSLint would catch this.